pkg/workload/schemachange/operation_generator.go

```diff
@@ -2373,6 +2373,16 @@ func (og *operationGenerator) insertRow(ctx context.Context, tx pgx.Tx) (stmt *o
 	if err != nil {
 		return nil, err
 	}
+	// If we aren't on 22.2 then disable the insert plugin, since 21.X
+	// can have queries fail due to an optimizer bug.
+	skipInserts, err := isClusterVersionLessThan(ctx, tx, clusterversion.ByKey(clusterversion.Start22_2))
+	if err != nil {
+		return nil, err
+	}
+	if skipInserts {
+		tableExists = false
+		tableName.SchemaName = "InvalidObjectName"
```

Comment:
How does these two lines disable selects? In particular, why is the second line (`tableName.SchemaName = "InvalidObjectName"`) needed?